ui/admin/src/app/libraries/library/library.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component } from '@angular/core';
+import { FormArray, FormGroup } from '@angular/forms';
+import { ActivatedRoute } from '@angular/router';
+import { Observable } from 'rxjs';
+
+import { environment } from '../../../environments/environment';
+
+import { LibraryFormService } from '../library-form.service';
+import { LibrariesService } from '../libraries.service';
+import { Library } from '../library';
+import { BrowserService } from '../../browser.service';
+
+@Component({
+  selector: 'libraries-library',
+  templateUrl: './library.component.html',
+  styleUrls: ['./library.component.scss']
+})
+export class LibraryComponent {
+
+  public library: Library;
+
+  public libForm: FormGroup;
+
+  constructor(
+    private browser: BrowserService,
+    private route: ActivatedRoute,
+    public librariesService: LibrariesService,
+    public libraryForm: LibraryFormService
+  ) {
+    this.librariesService.currentLibrary.subscribe(
+      library => {
+        this.library = library;
+        this.libraryForm.populate(library);
+        this.libForm = this.libraryForm.form;
```

Comment:
is it really used?

Reply:
Yes, i use this to subscribe to the library in main component.